source/beamline/BioXAS/BioXASBeamStatus.cpp

```diff
@@ -18,6 +40,18 @@ BioXASBeamStatus::~BioXASBeamStatus()
 
 }
 
+bool BioXASBeamStatus::isConnected() const
+{
+	bool connected = (
+				shutters_ && shutters_->isConnected() &&
+				valves_ && valves_->isConnected() &&
+				mirrorMaskState_ && mirrorMaskState_->isConnected() &&
+				monoMaskState_ && monoMaskState_->isConnected()
```

Comment:
Not as written. The Imaging beamline doesn't have an M1 mask or a mono mask--only Side and Main have those features.